lib/svrquery/protocol/titanfall/types.go

```diff
@@ -33,11 +38,21 @@ func (i Info) NumClients() int64 {
 	return int64(i.BasicInfo.NumClients)
 }
 
+// NumBotClients implements protocol.Responser.
```

Comment:
Is it a problem that this is zero for older versions vs not implementing the interface?